models/document-generation.ts

```diff
@@ -0,0 +1,426 @@
+/* eslint-disable camelcase */
+import { APIGatewayEventRequestContext } from "aws-lambda";
+
+export enum AwardType {
+  INITIAL_AWARD = "INITIAL_AWARD",
+  MODIFICATION = "MODIFICATION",
+}
+
+export enum DocumentType {
+  DESCRIPTION_OF_WORK = "DESCRIPTION_OF_WORK",
+}
+
+export enum PeriodType {
+  BASE = "BASE",
+  OPTION = "OPTION",
+}
+
+export enum PeriodUnit {
+  DAY = "DAY",
+  WEEK = "WEEK",
+  MONTH = "MONTH",
+  YEAR = "YEAR",
+}
+
+export enum TimeFrame {
+  NO_LATER_THAN = "NO_LATER_THAN",
+  NO_SOONER_THAN = "NO_SOONER_THAN",
+}
+
+export enum AddressType {
+  FOREIGN = "FOREIGN",
+  MILITARY = "MILITARY",
+  US = "US",
+}
+
+export enum Classification {
+  U = "U", // Unclassified
+  S = "S", // Secret
+  TS = "TS", // Top Secret
+}
+
+export enum ImpactLevel {
+  IL2 = "IL2",
+  IL4 = "IL4",
+  IL5 = "IL5",
+  IL6 = "IL6",
+}
+
+export enum InstanceLocation {
+  CSP = "CSP",
+  HYBRID = "HYBRID",
+  ON_PREMISE = "ON_PREMISE",
+}
+
+export enum TargetCspName {
+  CSP_A = "CSP_A",
+  CSP_B = "CSP_B",
+  CSP_C = "CSP_C",
+  CSP_D = "CSP_D",
+}
+
+export enum PricingModel {
+  ON_DEMAND = "ON_DEMAND",
+  PAY_AS_YOU_GO = "PAY_AS_YOU_GO",
+  RESERVED = "RESERVED",
+}
```

Comment:
Yesterday when doing the integration, I discovered that some of these `enums` in `document-generation.ts` (which is based on the schema I wrote) are not actually in the SNOW DB. These values are null on the SNOW side, so I think we should add a null option to this enum until we actually get it set up on the SNOW side.

```
export enum PricingModel {
  ON_DEMAND = "ON_DEMAND",
  PAY_AS_YOU_GO = "PAY_AS_YOU_GO",
  RESERVED = "RESERVED",
  null
}
```

Reply:
Fair, see my later comments where I indicate that it could be nullable inline.